.changes/next-release/feature-45f6e6a7-fc5b-4b14-89e2-0d9a2b138c65.json

```
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "/review: create General group for include_code_with_reference checkbox"
```

Should it be something like

```suggestion
  "description" : "/review: Moved "Include suggestions with code references" setting to General
```

---

And this is not specific to /review anymore, right?

---

Do you know what section should this be in?